apify-api/openapi/paths/datasets/datasets@{datasetId}@items.yaml

```diff
@@ -350,6 +350,15 @@ get:
       schema:
         type: boolean
         example: false
+    - name: view
+      in: query
+      description: |
```

Comment:
I'd mention that `overview` is more or less the default view and anything else is up to the Actor developer.